debian/homeserver.yaml

```diff
@@ -86,7 +86,7 @@ listeners:
   # For when matrix traffic passes through loadbalancer that unwraps TLS.
   - port: 8008
     tls: false
-    bind_address: ''
+    bind_address: 127.0.0.1
```

Comment:
Why do we want to default to localhost only?

Reply:
Well, my idea (at least for the ‘official’ Debian packages) was that making the server public should be a conscious decision of the system administrator, so that merely installing a package (possibly by accident) doesn’t result in a server listening on an external interface.